src/frontend/styles/vars.pcss

```diff
@@ -3,6 +3,8 @@
   --color-text-second: #5d6068;
   --color-line-gray: #E8E8EB;
   --color-link-active: #2071cc;
+  --color-link-hover: #F3F6F8;
+  --color-button-danger: #ff5159;
```

Comment:
looks like an irrelevant change

Reply:
Only `--color-button-danger` is irrelevant

Reply:
Fixed